specs/commandline/certificate.md

```diff
@@ -0,0 +1,247 @@
+# notation certificate
+
+## Description
+
+Use ```notation certificate``` command to add/list/delete certificates in a trust store. Update an existing certificate is not allowed since the thumbprint will be inconsistent, which results in a new certificate.
+
+The trust store is in the format of a directory in the filesystem as`x509/<type>/<name>/*.crt|*.cer|*.pem`. Currently two types of trust store are supported:
+
+* `Certificate Authority`: The directory name is `ca`.
+* `Signing Authority`: The directory name is `signingAuthority`
```

Comment:
Should we also account for tsa ?

https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-store

Reply:
According to this spec, TSA based timestamping will not be available in rc.1, so it is not listed here.